### PR TITLE
New page describing the planned end date of CTIS

### PR DESCRIPTION
**Pull request by capnrefsmmat**
**Prerequisites**:

- [x] Unless it is a documentation hotfix it should be merged against the `dev` branch
- [x] Branch is up-to-date with the branch to be merged with, i.e. `dev`
- [ ] Build is successful
- [x] Code is cleaned up and formatted

### Summary

Describes the end date of CTIS and the reasoning for it. **Not ready to be merged**; needs updating with a link to wherever UMD's equivalent page will be, so we can point to their schedule and reasoning.

Provided UMD is able to post their site, we'd like to deploy this around 9am ET on Friday, April 8.

### CODE REVIEW

File: docs/symptom-survey/end-of-survey.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+standards during the sampling period, all data will be maintained in a way that
+continues to preserve user privacy.
+
+All CTIS aggregate data will continue to be available after data collection
+stops on June 25, 2022 on [UMD‘s Global CTIS](https://covidmap.umd.edu/) and the
+Delphi Group at CMU’s US CTIS websites. Academic and nonprofit researchers may
+continue to [request access](./data-access.md) to non-public, non-aggregated
+survey data for their research, and current approved data users will be able to
+continue accessing the non-aggregated data until their current data use
+agreements (DUA) expire. Researchers currently with a fully executed DUA will
```
Comment (krivard) [inline docs/symptom-survey/end-of-survey.md:58]: ?
```suggestion
agreements (DUA) expire. Researchers currently holding a fully executed DUA will
```

**Review by krivard — DISMISSED** [review]
style nits optional 👍

**Comment by capnrefsmmat** [comment]
Thanks, merged your changes and updated with UMD's link. This should be good to go tomorrow morning.

**Review by krivard — APPROVED** [review]
👍

**Pull request merged**